src/components/FileUpload/Card.js

```diff
@@ -0,0 +1,24 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled, { css } from 'styled-components';
+
+const StyledCard = styled.div`
+  width: 100%;
+
+  ${(props) =>
+    props.state === 'error' &&
+    css`
+      border-color: #ff3b3b;
+    `};
+`;
+
+function Card(props) {
+  return <StyledCard {...props}> {props.children} </StyledCard>;
+}
+
+Card.propTypes = {
+  children: PropTypes.oneOfType([PropTypes.object, PropTypes.array]),
+  state: PropTypes.oneOf(['error', 'disabled'])
```

Comment:
Can't it be `ok`?

Reply:
You can add `ok` here and make it the default, thus you fix propTypes while retaining the current behavior